.travis.yml

```diff
@@ -3,8 +3,8 @@ language: ruby
 rvm:
 - 2.5.7
 - 2.6.5
-sudo: false
 cache:
   bundler: true
 script:
 - bundle exec rspec
+- test -z "$(gofmt -l manageiq-operator)"
```

Comment:
Obviously it's there, but why should we expect `gofmt` to be available on our test image?